.travis.yml

```diff
@@ -24,6 +24,9 @@ jobs:
 
   - stage: deploy
     install: skip
+    before_install:
+      - sudo apt-get update
+      - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
```

Comment:
I presume we need this to get 17.06+ for multistage builds?

Reply:
sounds about right.  not sure about the specific version, but travis was using an old version that couldn't parse the "FROM blah AS blah" format